interpreter/terminal_interface/start_terminal_interface.py

```diff
@@ -689,3 +690,11 @@ def start_terminal_interface(interpreter):
     interpreter.in_terminal_interface = True
 
     interpreter.chat()
+
+
+def main():
+    interpreter = OpenInterpreter()
+    try:
+        start_terminal_interface(interpreter)
+    except KeyboardInterrupt as e:
+        print(e)
```

Comment:
I believe `e` will always be an empty string. Do we want to revert to `Exited.` or some other message to show the user?

Reply:
It does print something, no?
Anyways, I added more Error handling and a finally:

Reply:
It didn't print anything in my testing and chatGPT says it'll always be empty. I'm no python expert though. Did it print a message in your testing?